docs/resources/azure_aks_cluster.md

```diff
@@ -0,0 +1,93 @@
+---
+title: About the azure_aks_cluster Resource
+platform: azure
+---
+
+# azure_aks_cluster
+
+Use the `azure_aks_cluster` InSpec audit resource to test properties of an Azure AKS Cluster.
+
+## Azure REST API version, endpoint and http client parameters
+
+This resource interacts with api versions supported by the resource provider.
+The `api_version` can be defined as a resource parameter.
+If not provided, the latest version will be used.
+For more information, refer to [`azure_generic_resource`](azure_generic_resource.md).
+
+Unless defined, `azure_cloud` global endpoint, and default values for the http client will be used .
+For more information, refer to the resource pack [README](../../README.md). 
+
+## Availability
+
+### Installation
+
+This resource is available in the [InSpec Azure resource pack](https://github.com/inspec/inspec-azure). 
+For an example `inspec.yml` file and how to set up your Azure credentials, refer to resource pack [README](../../README.md#Service-Principal).
+
+## Syntax
+
+An `azure_aks_cluster` resource block identifies an AKS Cluster by `name` and `resource_group`.
+```ruby
+describe azure_aks_cluster(resource_group: 'example', name: 'ClusterName') do
+  it { should exist }
+end
+```
+## Parameters
+
+| Name                           | Description                                                                       |
+|--------------------------------|-----------------------------------------------------------------------------------|
+| resource_group                 | Azure resource group that the targeted resource resides in. `MyResourceGroup`     |
+| name                           | Name of the AKS cluster to test. `ClusterName`                                      |
+| resource_id                    | The unique resource ID. `/subscriptions/{subscriptionId}/resourceGroups/{resourceGroup}/providers/Microsoft.ContainerService/managedClusters/{ClusterName}` |
+
+Either one of the parameter sets can be provided for a valid query:
```

Comment:
Based on this, am I right to assume that this is also valid?
```
describe azure_aks_cluster(resource_id: 'example') do
   it { should exist }
 end
```
If so, it would be good to provide this an example also.